src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Condition/IsCancellationCompletedPlugin.php

```diff
@@ -24,6 +24,8 @@ class IsCancellationCompletedPlugin extends AbstractPlugin implements ConditionI
     public const CANCEL_TRANSACTION_ACCEPTED = AfterPayConfig::API_TRANSACTION_OUTCOME_ACCEPTED;
 
     /**
+     * {@inheritDoc}
+     *
```

Comment:
Add a specification to a method:
- Checks if the cancel operation was successfully accepted for the order.